semiosis/agents/anthropic_agent.py

```
@@ -0,0 +1,412 @@
+        return AgentResponse(
+            output="",
+            logprobs={},
+            metadata={
+                "error": "Maximum retries exceeded",
+                "model": self.model,
+                "attempts": self.max_retries + 1
+            },
+            cost=0.0
+        )
+    
+    def extract_logprobs(self, query: str, response: str, context: Optional[str] = None) -> Dict[str, float]:
```

Ruff (0.14.5)

Unused method argument: `query`

(ARG002)